rcl/test/rcl/client_fixture.cpp

```diff
@@ -24,6 +24,33 @@
 #include "../memory_tools/memory_tools.hpp"
 #include "../scope_exit.hpp"
 #include "rcl/error_handling.h"
+#include "rcl/graph.h"
+
+bool
```

Comment:
Since the function only return either "true" or "false" I don't think the type should be `rcl_ret_t` which implies multiple other values are possible.